src/fmu/dataio/providers/objectdata/_xtgeo.py

```diff
@@ -82,16 +73,28 @@ def get_bbox(self) -> BoundingBox2D | BoundingBox3D:
             ymax=float(self.obj.ymax),
         )
 
+    def get_spec(self) -> SurfaceSpecification:
+        """Derive data.spec for xtgeo.RegularSurface."""
+        logger.info("Get spec for RegularSurface")
```

Comment:
These `get_spec`s did not get changed, just reordered alphabetically among other getters.